python/doc/examples/calibration/bayesian_calibration/plot_imh_python_distribution.py

```diff
@@ -0,0 +1,122 @@
+"""
+Sampling from an unnormalized probability density
+=================================================
+We sample from an unnormalized probability density
+using Metropolis-Hastings algorithms.
+  .. math::
+     f(x) = \\frac{1}{2} (2 + \sin(x)^2) \\exp \\left[- \\left(2 + \\cos(3x)^3 + \\sin(2x)^3 \\right) x \\right] \\mathbf{1}_{[0, 2 \\pi]}(x).
```

Comment:
```suggestion

.. math::
   f(x) = \\frac{1}{2} (2 + \sin(x)^2) \\exp \\left[- \\left(2 + \\cos(3x)^3 + \\sin(2x)^3 \\right) x \\right] \\mathbf{1}_{[0, 2 \\pi]}(x).

```

I assume the "indentation problem" from CI has something to do with this block. Try this version.